### ISSUE TITLE
Any issue with Dunghill parser?

### ISSUE THREAD

**Issue by ErezDasa** [JMousqueton/ransomware.live#67]
Seems like the last victim were not detected (supply technologies llc)

**Comment by JMousqueton** [comment]
exact, I've correct it for now but i modify permanently the parser. 
Thank you for noticing.
I'll keep this Issue open till I'll modify the parser

**Comment by JMousqueton** [comment]
Fixed
